cranelift/codegen/src/ir/extfunc.rs

```diff
@@ -424,7 +424,5 @@ mod tests {
         assert_eq!(sig.to_string(), "(i32) -> f32 windows_fastcall");
         sig.params.push(AbiParam::new(I32.by(4).unwrap()));
         assert_eq!(sig.to_string(), "(i32, i32x4) -> f32 windows_fastcall");
-        sig.returns.push(AbiParam::new(B8));
```

Comment:
Please push another type instead. I believe it tests multi return function signatures.

Reply:
Thanks for catching that, I've switched it to `i8`.